src/plugins/parser/divclass.js

```diff
@@ -72,45 +72,47 @@ module.exports = function divclass() {
   const methods = Parser.prototype.blockMethods;
   const restorationMethods = Parser.prototype.restorationMethods;
 
-  restorationMethods.divclass = function(add, nodes, content, children) {
```

Comment:
Why was `redactedLink` moved to `remark-redactable` while `divclass` stayed here?

Reply:
mostly because it had previously been added to `remark-redactable` in order to enable easier testing.

I think the actual correct place for both of these plugins is https://github.com/code-dot-org/remark-plugins, but I didn't want to move them there as part of this change because it's already gotten big enough.